src/documentation/Usage/ToolRecommendations.stories.mdx

```diff
@@ -39,6 +39,8 @@ Please have a look at the [Map guidelines](Design|Guidelines).
 
 ~~[redux-form](https://github.com/erikras/redux-form) works with React Redux to enable an html form in React to use Redux to store all of its state.~~
 
+[react-phone-input-2](https://www.npmjs.com/package/react-phone-input-2) is a customizable phone input library with auto formatting.
```

Comment:
https://www.npmjs.com/package/react-phone-number-input Is also nice. Maybe we can investigate which one is better.